### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by titor-oopart**

### CODE REVIEW

File: src/main/java/core/selenium/webdriver/ConnectionFactory.java [hunk 1]
```diff
@@ -22,14 +22,14 @@
...
-    public WebDriverConfig webDriverConfig;
+    private WebDriverConfig webDriverConfig;
     private WebDriver driver;
 
     /**
      * This method chosse the browser type accord of the properties.
      *
      * @param browser - Type of Browser for the test.
-     * @return
+     * @return driver
```
Comment (MelissaRoman06) [inline src/main/java/core/selenium/webdriver/ConnectionFactory.java]: Please add documentation.

File: src/test/resources/features/Search.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  Scenario: Search a job by keyword
+    Given the user goes principal page
+    When user search a specific keyword
+    Then the result conteins in his category this keyword
+
+  Scenario: Advance search by city
+    Given the user goes principal page
+    When user search by a specific city
+    Then the result conteins the city in the location field
```
Comment (MelissaRoman06) [inline src/test/resources/features/Search.feature]: Missepelling in "conteins".

File: src/test/resources/features/Login.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Feature: Login in an account
+
+  Scenario: login in trabajopolis
```
Comment (MelissaRoman06) [inline src/test/resources/features/Login.feature]: Please use capital letter at the beginning

File: src/main/java/core/selenium/webdriver/ConnectionFactory.java (hunk 1 above)
Comment (titor-oopart) [reply src/main/java/core/selenium/webdriver/ConnectionFactory.java]: done

File: src/test/resources/features/Search.feature (hunk 2 above)
Comment (titor-oopart) [reply src/test/resources/features/Search.feature]: done

File: src/test/resources/features/Login.feature (hunk 3 above)
Comment (titor-oopart) [reply src/test/resources/features/Login.feature]: done

**Pull request merged**